### ISSUE TITLE
Can't add dev dependency with `poetry add --dev` if [tool.poetry.dev-dependencies] doesn't exist

### ISSUE THREAD

**Issue by moigagoo** [sdispater/poetry#61]
Here's the command I'm running:

```shell
poetry add --dev mkdocs

  [KeyError]


add [-D|--dev] [--optional] [--allow-prereleases] [--dry-run] [--] <name> (<name>)…
```

Also tried `poetry add mkdocs --dev`, with the same result.

Using poetry 0.8.5 on Windows.

**Comment by sdispater** [comment]
Could you execute the command again in verbose mode (the `-v` option)?

**Comment by moigagoo** [comment]
@sdispater Strangely enough, I does work now, without any issues ¯\_(ツ)_/¯

All I've done is run it in ConEmu this time instead of FluentTerminal. But it's the same PowerShell in both cases.

**Comment by moigagoo** [comment]
OK, I located the issue source. The command fails if `[tool.poetry.dev-dependencies]` is missing in pyproject.toml.

**Comment by sdispater** [comment]
This should be fixed in commit 2e36b03. This was made possible by changing the TOML library used by Poetry.

It will land in the next bugfix release.

**Comment by sdispater** [comment]
Release `0.11.3` is out!
